plugin/kubernetes/handler.go

```diff
@@ -53,6 +53,39 @@ func (k Kubernetes) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns.M
 			break
 		}
 		fallthrough
+	case dns.TypeAXFR, dns.TypeIXFR:
+		k.Transfer(ctx, state)
+		/*
```

Comment:
Why is all this commented out code still here?

Reply:
Poor memory. I'll get it cleaned up